src/turbomind/utils/Tensor.cc

```diff
@@ -22,7 +22,14 @@
 #include "stdlib.h"
 #include <cuda_fp16.h>
 #include <cuda_runtime_api.h>
+#ifdef _MSC_VER
+#define NOMINMAX
+#include "src/turbomind/windows/dirent.h"
```

Comment:
We don't need this. Switch to cross-platform library utility [std::filesystem::directory_iterator](https://en.cppreference.com/w/cpp/filesystem/directory_iterator) instead of adding 3rdparty dependencies.